en, zh: prompt user not to modify names of Secret objects. (#1272)
pingcap/docs-tidb-operator: en/enable-tls-for-mysql-client.md

1. [Issue two sets of certificates](#issue-two-sets-of-certificates-for-the-tidb-cluster): a set of server-side certificates for TiDB server, and a set of client-side certificates for MySQL client. Create two Secret objects, `${cluster_name}-tidb-server-secret` and `${cluster_name}-tidb-client-secret`, respectively including these two sets of certificates.

    > **Note:**
    >

>     > Secret objects can't use any other name.

Reviewer: ```suggestion
    > The Secret objects you created must follow the above naming convention. Otherwise, the TiDB cluster cannot be deployed successfully.
```

>     > The Secret objects you created must follow the above naming convention. Otherwise, the TiDB cluster cannot be deployed successfully.

Reviewer: ```suggestion
    > The Secret objects you created must follow the above naming convention. Otherwise, the deployment of the TiDB cluster will fail.
```
